macros/cssxref.ejs

```diff
@@ -73,7 +68,7 @@ if (!$2) {
     //if (!thisPage) thisPage = wiki.getPage(url); // lazy evaluation // Disabled to test performance (we get a lot of Kumascript errors)
 
     if (thisPage && thisPage.summary) {
-        summary = mdn.escapeQuotes(thisPage.summary);
+        summary = kuma.htmlEscape(thisPage.summary);
```

Comment:
Yeah, I missed that escapeQuotes() also removes HTML tags. I'll change this back then.